cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml

```diff
@@ -10,7 +10,9 @@ resources:
 - ../../bundles/xdmod-reader
 - feature/odf
 - feature/custom-routes
-- ../../base/core/namespaces/openshift-gitops
```

Comment:
Removing this namespace doesn't appear to be related to the rest of this PR. Was this intentional?

Reply:
Thanks @larsks , I put it back in.